extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -1063,3 +1063,116 @@ async def get(self, objectId: str = None):
         except Exception:
             _err = traceback.format_exc()
             return self.error(f"failure: {_err}")
+
+
+class ZTFAlertsByCoordsHandler(ZTFAlertHandler):
+    @auth_or_token
+    def get(self):
+        """
+        ---
+        single:
+          description: Query Kowalski for alerts near specified coords
+          parameters:
+            - in: query
+              name: ra
+              required: true
+              schema:
+                type: float
+            - in: query
+              name: dec
+              required: true
+              schema:
+                type: float
+            - in: query
+              name: radius
+              required: true
+              schema:
+                type: float
+          responses:
+            200:
+              description: retrieved alert(s)
+              content:
+                application/json:
+                  schema:
+                    allOf:
+                      - $ref: '#/components/schemas/Success'
+                      - type: object
+                        properties:
+                          data:
+                            type: array
+                            items:
+                              type: object
+            400:
+              content:
+                application/json:
+                  schema: Error
+        """
+        ra = self.get_query_argument("ra", None)
+        dec = self.get_query_argument("dec", None)
+        radius = self.get_query_argument("radius", None)
+        if ra is None:
+            return self.error("Missing required parameter: ra")
+        if dec is None:
+            return self.error("Missing required parameter: dec")
+        if radius is None:
+            return self.error("Missing required parameter: radius")
+        try:
+            ra = float(ra)
+            dec = float(dec)
+            radius = float(radius)
+        except ValueError:
+            return self.error("Invalid (non-float) value provided.")
+        streams = self.get_user_streams()
```

Comment:
It seems like `get_user_streams()` does the same thing as `self.associated_user_object.streams`:

```python
class ZTFAlertHandler(BaseHandler):
    def get_user_streams(self):

        streams = (
            DBSession()
            .query(Stream)
            .join(StreamUser)
            .filter(StreamUser.user_id == self.associated_user_object.id)
            .all()
        )
        if streams is None:
            streams = []

        return streams
```

Is there an advantage to using this over simply `self.associated_user_object.streams`?

Reply:
Ask @dmitryduev -- I'm just mirroring what he's already implemented in other handlers here (the function is not new to this PR)

Reply:
Resolving as out of scope